woocommerce/woocommerce
# Add combobox and extra status information to the live branches menu in WooCommerce Beta Tester

### All Submissions:


### Changes proposed in this Pull Request:

Inspired by the Jetpack Beta menu this adds menus that show:

1. The currently running version of WooCommerce, and some basic version info if its not a live branch.
2. A combobox menu thanks to the work of @jonathansadowski in #35958 that lets you select new versions to install and activate
3. a list of other versions you already have installed which you can activate if you wish to easily switch between live branches you already downloaded earlier.

This is how it looks:

<img width="1120" alt="Screenshot 2022-12-22 at 3 35 32 PM" src="https://user-images.githubusercontent.com/1281828/209043173-47c91db9-7cfe-4197-b4a5-529bf8a8adfc.png">


Also note that the state now properly updates so you don't have a situation where multiple items would say they are activated which was something I wanted to enhance in #36072 






### How to test the changes in this Pull Request:

1. Checkout this branch locally
2. Have an env with beta tester plugin and Woo installed
3. Go to WooCommerce -> Live Branches 
4. You should see 2 panels, one shows the currently running version of Woo and another has a combobox
5. open the combobox and choose a live branch
6. click install and activate
7. After some time the new version should have installed and activated. The version information panel should now change to show it is running your live branch.
8. Now choose another combobox option
9. Click install and activate
10. the version should be switched after this finishes.
11. Refresh the page and you should see your previously activated branch under "installed branches" in a 3rd panel.


### Other information:




### FOR PR REVIEWER ONLY:
